docs/source/private_data_tutorial.rst

```diff
@@ -731,16 +745,27 @@ When successful, should see something similar to the following result:
 Query private data Org2 is not authorized to
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Peers in Org2 do not have the Org1 asset1 ``appraisedValue`` private data in their side database.
-When they try to query for this data, they get back a hash of the key matching
-the public state but will not have the private state.
+Peers in Org2 do not have the Org1 asset1 ``appraisedValue`` private data in their
```

Comment:
```suggestion
Peers in Org2 do not have the Org1 owned asset1's ``appraisedValue`` private data in their
```

Reply:
Peers in Org2 do not have the ``appraisedValue`` of the asset1 owned by Org2 in their
side database.